nml/expression/storage_op.py

```diff
@@ -121,3 +121,6 @@ def supported_by_actionD(self, raise_error):
         if raise_error:
             raise generic.ScriptError("{}() may only be used inside switch-blocks".format(self.name), self.pos)
         return False
+
+    def is_read_only(self):
+        return (not self.info['store']) and self.register.is_read_only() and (self.value.is_read_only() if self.value is not None else True)
```

Comment:
might consider pulling the `self.value` bit into a separate variable

Reply:
Given the line length I'd suggest splitting this over multiple lines:

```
if self.value is None:
  return True
return ....
```

Reply:
After more thinking I noticed `self.value` can be ignored, as it's set only on write.